doc/source/management/data_at_rest_encryption.rst

```diff
@@ -412,8 +351,10 @@ the required options.
 .. code-block:: text
 
    [mysqld]
-   basedir=/dev/shm/pxc57
-   datadir=/dev/shm/pxc57/copy_mig
-   log-error=/dev/shm/pxc57/logs/copy_mig.err
+   basedir=/dev/shm/pxc80
+   datadir=/dev/shm/pxc80/copy_mig
+   log-error=/dev/shm/pxc80/logs/copy_mig.err
    socket=/tmp/copy_mig.sock
    port=16400
+
+.. include:: ../.res/replace.txt
```

Comment:
We should expand this documentation to discuss how we perform transition keys during SST and discuss SST / DARE interactions.